desktop/assets/texts/ru-RU/preferences_dialog.ftl

```diff
@@ -17,7 +17,7 @@ log-filename-pattern-with-timestamp = С временной меткой
 storage-backend = Место сохранения файла
 storage-backend-disk = Диск
 storage-backend-memory = Память
-recent-limit = Недавний лимит
+recent-limit = Лимит недавних
```

Comment:
Perhaps this source string could also use some context or explanation, as it was misunderstood the first time (as "the limit is recent", as opposed to "the limit of recent things").

Reply:
Added context on Crowdin.